editor/src/components/user-bar.tsx

```diff
@@ -155,7 +179,20 @@ const MultiplayerAvatar = React.memo(
             cursor: 'pointer',
           }}
         >
-          {props.name}
+          {when(picture == null, props.name)}
+          {when(
```

Comment:
nit: maybe an `unless( picture == null,` is easier to read, maybe not, I'm not really sure, why am I even typing

Reply:
https://github.com/concrete-utopia/utopia/pull/4502/commits/1859cff20c60bd00d99bd280379904dbf6be40f7